### PR TITLE
Add darwinia smart chain part

### PR DESCRIPTION
**Pull request by boundless-forest**

### CODE REVIEW

File: docs/chains/darwinia-chains.md [hunk 1]
```diff
@@ -13,10 +13,6 @@ Darwinia Chain (DC) is designed to be the primary chain for tokens and governanc
 DC is a public chain that can operate independently with its own consensus and security model, with its core business and application services, including cross-chain functionality of each application chain, controlled by 
 DC itself.
 
-#### Pangoro
```
Comment (wuminzhe) [inline docs/chains/darwinia-chains.md:16]: Why delete the pangoro here? This doc contains all darwinia chains including the testnets.

File: docs/chains/darwinia-smart-chain.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- Token Decimal: `18`
+- ChainId: `45`
+- Block Explorer URL: `https://pangoro.subscan.io/`
+- HTTP Endpoints:
+    - `https://pangoro-rpc.darwinia.network`
+- WSS Endpoints:
+    - `wss://pangoro-rpc.darwinia.network`
+
+
+### Pre-compiles
```
Comment (wuminzhe) [inline docs/chains/darwinia-smart-chain.md]: Precompiled Contracts or Precompile, according to EIP name convention.

Comment (boundless-forest) [reply docs/chains/darwinia-smart-chain.md]: Make sense. I'll update this later.

File: docs/chains/darwinia-chains.md (hunk 1 above)
Comment (boundless-forest) [reply docs/chains/darwinia-chains.md:16]: image

Just follow the Crab network doc home page outline. Adding a separate section about the test chain here looks a little complicated. So I put the information about the Pangoro Smart Chain in the Darwinia Smart Chain introduction.

**Comment by boundless-forest** [comment]
@wuminzhe Please review again!

**Review by wuminzhe — APPROVED** [review]

**Pull request merged**